src/ProcessManager/ChromeManager.php

```diff
@@ -77,7 +77,7 @@ private function findChromeDriverBinary(): string
         }
     }
 
-    private function getDefaultArguments(): array
+    public function getDefaultArguments(): array
```

Comment:
Why this change? If it is just for https://github.com/dunglas/panthere/pull/15/files#diff-0b7a157b54171e8b04337166e6da76d3R47, it is not worth it, just copy them.